adafruit_tcs34725.py

```diff
@@ -311,7 +311,11 @@ def _temperature_and_lux_dn40(self):
         lux = G1 / CPL
 
         # CT Calculations (DN40 3.4)
-        CT = CT_Coef * B2 / R2 + CT_Offset
+        # Prevent divide by zero errors. if r2 = 0 use a really small r2 to print offset
+        if R2 > 0:
+            CT = CT_Coef * B2 / R2 + CT_Offset
+        else:
+            CT = CT_Coef * B2 / 0.001 + CT_Offset
```

Comment:
Instead of copying and changing the computation line please invert the if to check R2 == 0 and then set R2 to 0.001 instead.

Reply:
How about limiting all the values in the initial computation of R2, G2, and B2?
```python
R2 = max(0.001, R - IR)
G2 = max(0.001, G - IR)
B2 = max(0.001, B - IR)
```
and also define a const up to for the `0.001` value